cdap-docs/examples-manual/source/examples/wikipedia-data-pipeline.rst

```diff
@@ -0,0 +1,134 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :description: Cask Data Application Platform Wikipedia Pipeline Application
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _examples-wikipedia-data-pipeline:
+
+==================
+Wikipedia Pipeline
+==================
+
+A Cask Data Application Platform (CDAP) example demonstrating a typical data processing pipeline using CDAP Workflows.
+
+Overview
+========
+
+This example demonstrates a CDAP application performing analysis on Wikipedia data using MapReduce and Spark programs
+running within a CDAP Workflow - *WikipediaPipelineWorkflow*.
+
+This example can be run in both online and offline modes. In the online mode, the custom action *DownloadWikiDataAction* reads the *pageTitleStream*

```

Comment:
To make it clearer, I would start a new paragraph before each mode, because it becomes easier to see what's what. You can bullet them if you want.